### ISSUE TITLE
Some additional info for ASUS TUF Gaming FX505DY

### ISSUE THREAD

**Issue by RobertMueller2** [T-vK/gpu-passthrough.com#4]
Hi,

I've had a look at passing through the dGPU for this model to Windows 10. At some point, after a combination of host reboots/hibernates and VM reboots, Windows did not show error 43. I reached this case twice but sadly, not consistently in any way. Unfortunately, I have not managed to setup looking glass or steam during this time, so it's not possible to say if it REALLY worked.

Something else I learned on the way:

It's not possible to read the dGPU ROM through sysfs, and neither with GPU-Z in native Windows. It is however possible to extract the ROM from Asus's firmware using UEFIextract. Interestingly, as of version 314, the GPU ROM is legacy only and does not appear to contain a UEFI part. (Checked with rom-parser.)

So in order to use the ROM, the VM would need OVMF-with-CSM or SeaBIOS. I have not used it for the 2 instances of success -- but it *might* be useful for more consistent results.

R.

**Comment by azrulhaifan** [comment]
hey, have you succeeded passthrough dGPU on this laptop ? i have same model, but even i can't extract vbios from latest 315 bios.

**Comment by RobertMueller2** [comment]
No, I stopped bothering after a while. The biggest annoyance for me was that switching between vfio and amdgpu often caused crashes. I couldn't stand rebooting all the time and effectively that was worse than dual boot. :-p

Now there's been a few kernel versions and a number of amdgpu fixes in the meantime, perhaps it's a good time to have another go at it.

**Comment by skinatro** [comment]
hey I have been all trying to pass through the gpu, I followed the guide by the passthrough post and I am using virt-manager but whenever I use the gpu , it says,graphical console not running or something similar to it. what did you do to passthrough the gpu? @RobertMueller2

**Comment by RobertMueller2** [comment]
I'd really have to have a closer look at this again. IIRC once you got lookingglass working, you would use the lookingglass client instead of virt-viewer. (But I never reached that point yet.) Before you got lookingglass going, you have to figure out how to access the VM. This might be a starting point https://github.com/T-vK/MobilePassThrough or you could add a QXL and Spice Display temporarily and use virt-viewer, or perhaps use rdp.

Apologies for having to tell this from top of my head, I have not looked at it pretty much since the first posting and right now I'm swamped with projects. I'll get back when I find some time.
